pydrive2/test/README.rst

```diff
@@ -1,13 +1,30 @@
-Instructions
-------------
-
--  Install timeout-decorator and futures (used for parallel upload and
-download testing).
--  Create *configs* and *credentials* folder in working directory.
--  Put *client_secrets.json* file from API console into *configs* folder.
--  Put *clients_secrets_local.json* from API console, for native device in
-*configs* folder.
--  Put different *a.png* and *b.png* file in working directory.
--  Replace {{ }} sections in *settings/test2.yaml* with the relevant sections
-from your config file.
--  For ServiceAuth test, place PKCS12 file in working directory.
+Run tests locally
+-----------------
+
+1. Setup a Google service account for your Google Cloud Project:
+    - Sign into the `Google API Console <https://console.developers.google.com>`_
+    - Select or `Create a new <https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project>`_ project.
+    - `Enable the Drive API <https://developers.google.com/drive/api/v2/about-sdk>`_ from the **APIs & Services** **Dashboard** (left sidebar), click on **+ ENABLE APIS AND SERVICES**. Find and select the "Google Drive API" in the API Library, and click on the **ENABLE** button.
+    - Go back to **IAM & Admin** in the left sidebar, and select **Service Accounts**. Click **+ CREATE SERVICE ACCOUNT**, on the next screen, enter **Service account name** e.g. "PyDrive tests", and click **Create**. Select **Continue** at the next **Service account permissions** page, click at **+ CREATE KEY**, select **P12** and **Create**. Save generated **.p12** key file at your local disk.
```

Comment:
will it set some default Owner permissions?

Reply:
It works fine without any assigned Roles (permissions). By default it doesn't assign any permissions. As far as Google Cloud Project has access to Google Drive API, service account will able to use Drive API to access service account's own GDrive disk space.

Reply:
Even configured OAuth Consent screen is not required to use Service Account.

Reply:
👍